.changeset/smart-owls-trade.md

```diff
@@ -0,0 +1,53 @@
+---
+'@sveltejs/adapter-cloudflare-workers': major
+---
+
+feat: use the new Workers Static Assets feature from Cloudflare
+
+This changes the adapter to stop using the old Workers Sites (kv-asset-handler) approach.
+Instead, making use of the new Workers Static Assets feature, which is embedded into Cloudflare natively.
+
+Also this change removes the extra esbuild step that was being run inside the adapter, relying upon Wrangler to do the bundling.
+The extra esbuild step required a hardcoded list of Node.js compatible modules.
+This is no longer needed since Wrangler now manages all of that.
+
+## Breaking changes and migration
+
+- This version of the adapter requires Wrangler version 3.87.0 or later.
+
+  Run `npm add -D wrangler@latest` (or similar) in your project to update Wrangler.
+- The user's Wrangler configuration (`wrangler.toml`) must be migrated from using Workers Sites to using Workers Assets.
+
+  Previously a user's `wrangler.toml` might look like:
```

Comment:
is there any migration guide on the cloudflare site we could point to instead of including the details here? most of our changelog entries are fairly minimal